src/components/ProjectSettingsTabs/AliasRow.tsx

```diff
@@ -71,90 +67,119 @@ export const AliasRow: SpruceFormProps["ObjectFieldTemplate"] = ({
   const makeId = (fieldName: string): string =>
     `${idSchema.$id}-${fieldName}-field`;
 
-  return (
+  const contents = (
+    <>
+      {alias.content}
+      <TaskRegexContainer>
+        <StyledSegmentedControl
+          label="Variant"
+          id="variant-input-control"
+          value={variantInput}
+          onChange={(value) => {
+            setVariantInput(value);
+            clearForm(value);
+          }}
+        >
+          <SegmentedControlOption
+            value={VariantInput.Tags}
+            disabled={isDisabled}
+            aria-controls={makeId(VariantInput.Tags)}
+          >
+            Tags
+          </SegmentedControlOption>
+          <SegmentedControlOption
+            value={VariantInput.Regex}
+            disabled={isDisabled}
+            aria-controls={makeId(VariantInput.Regex)}
+          >
+            Regex
+          </SegmentedControlOption>
+        </StyledSegmentedControl>
+        {variantInput === VariantInput.Tags ? (
+          <div data-cy="variant-tags-field" id={makeId(VariantInput.Tags)}>
+            {variantTags.content}
+          </div>
+        ) : (
+          <div data-cy="variant-field" id={makeId(VariantInput.Regex)}>
+            {variant.content}
+          </div>
+        )}
+      </TaskRegexContainer>
+      <TaskRegexContainer>
+        <StyledSegmentedControl
+          label="Task"
+          id="task-input-control"
+          value={taskInput}
+          onChange={(value) => {
+            setTaskInput(value);
+            clearForm(value);
+          }}
+        >
+          <SegmentedControlOption
+            value={TaskInput.Tags}
+            disabled={isDisabled}
+            aria-controls={makeId(TaskInput.Tags)}
+          >
+            Tags
+          </SegmentedControlOption>
+          <SegmentedControlOption
+            value={TaskInput.Regex}
+            disabled={isDisabled}
+            aria-controls={makeId(TaskInput.Regex)}
+          >
+            Regex
+          </SegmentedControlOption>
+        </StyledSegmentedControl>
+        {taskInput === TaskInput.Tags ? (
+          <div data-cy="task-tags-field" id={makeId(TaskInput.Tags)}>
+            {taskTags.content}
+          </div>
+        ) : (
+          <div data-cy="task-field" id={makeId(TaskInput.Regex)}>
+            {task.content}
+          </div>
+        )}
+      </TaskRegexContainer>
+    </>
+  );
+
+  return useExpandableCard ? (
+    <ExpandableCard
```

Comment:
do we want any spacing between the expandable cards? (the top and bottom of each card touch)

<img width="1141" alt="Screen Shot 2022-02-14 at 1 13 40 PM" src="https://user-images.githubusercontent.com/47064971/153922480-0dc3c281-5c05-4ba5-847b-f2e915059ff1.png">